Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,63 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            char again = 'y';
+
+            while (again == 'y')
+            {
+                RequestInput("first name");
+                string firstName = Console.ReadLine();
+
+                RequestInput("last name");
+                string surname = Console.ReadLine();
+
+                RequestInput("age in years");
+                int age = Convert.ToInt32(Console.ReadLine());
+
+                RequestInput("weight in kilograms");
+                double weight = Convert.ToDouble(Console.ReadLine());
+
+                RequestInput("height in centimeters");
+                double height = Convert.ToDouble(Console.ReadLine());
+
+                ReportBasicInfo(firstName, surname, age, weight, height);
+
+                ReportBMI(firstName, weight, height);
+
+                again = RequestUserChoiceToRepeatOrNot();
+            }
+        }
+
+        private static void RequestInput(string prompt)
+        {
+            Console.Write("Please enter the person's " + prompt + ": ");
         }
+
+        private static void ReportBasicInfo(string firstName, string surname, int age, double weight, double height)
+        {
+            Console.WriteLine("\n" + firstName + " " + surname +
+                " is " + age + " years old, " +
+                "their weight is " + weight + " kg " +
+                "and their height is " + height + " cm.");
+        }
+
+        private static void ReportBMI(string firstName, double weight, double height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'ReportBMI' to match pascal case naming rules, consider using 'ReportBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4828880)